packages/keyValueCache/src/InMemoryLRUCache.ts

```diff
@@ -32,7 +34,7 @@ export class InMemoryLRUCache<V extends {} = string>
     return 1;
   }
 
-  async set(key: string, value: V, options?: KeyValueCacheSetOptions) {
+  async set(key: string, value: V, options?: SO) {
     if (options?.ttl) {
       this.cache.set(key, value, { ttl: options.ttl * 1000 });
     } else {
```

Comment:
It seems weird to update the types here if we aren't passing through the rest of the options to the `.set` call. With this change: as an `InMemoryLRUCache` consumer, I can provide my own `SO` interface, safely call `.set` with options using the interface that I defined (`SO`), but that options object won't be passed to the underlying cache (the surprising part).

Reply:
@trevor-scheer Good catch! I've updated the `InMemoryLRUCache` to use the `LRUCache.SetOptions` type for the set options, and am always forwarding the set options through to the underlying LRU cache `set()` function.